core/src/io/anuke/mindustry/net/EditLog.java

```diff
@@ -30,6 +34,15 @@ public static EditLog valueOf(String string) {
 						   EditAction.valueOf(parts[3]));
 	}
 	
+	public static Array<EditLog> logsFromTile(int x, int y) {
```

Comment:
This whole method can be changed to `netServer.admins.getEditLogs().get(x + y * world.width(), new Array<>());`.  (Why are you checking each entry sequentially? This defeats the whole purpose of using a map.)